tests/tensorflow/r2.13/common.libsonnet

```diff
@@ -0,0 +1,327 @@
+          global_batch_size: 16384,
+        },
+        validation_data: {
+          input_path: '$(CRITEO_DATA_DIR)/eval/*',
+          global_batch_size: 16384,
+        },
+        model: {
+          num_dense_features: 13,
+          bottom_mlp: [512, 256, 64],
+          embedding_dim: 64,
+          top_mlp: [1024, 1024, 512, 256, 1],
+          vocab_sizes: [
+            39884406,
+            39043,
+            17289,
+            7420,
+            20263,
+            3,
+            7120,
+            1543,
+            63,
+            38532951,
+            2953546,
+            403346,
+            10,
+            2208,
+            11938,
+            155,
+            4,
+            976,
+            14,
+            39979771,
+            25641295,
+            39664984,
+            585935,
+            12972,
+            108,
+            36,
+          ],
+        },
+      },
+      trainer: {
+        use_orbit: true,
+        validation_interval: 90000,
+        checkpoint_interval: 270000,
+        validation_steps: 5440,
+        train_steps: 256054,
+        optimizer_config: {
+          embedding_optimizer: 'SGD',
+          lr_config: {
+            decay_exp: 1.6,
+            decay_start_steps: 150000,
+            decay_steps: 136054,
+            learning_rate: 30,
+            warmup_steps: 8000,
+          },
+        },
+      },
+    },
+    command: [
+      'python3',
+      'official/recommendation/ranking/train.py',
+      '--params_override=%s' % (std.manifestYamlDoc(self.paramsOverride) + '\n'),
+      '--model_dir=$(MODEL_DIR)',
+    ],
+  },
+  imagenet:: {
+    scriptConfig+: {
+      trainFilePattern: '$(IMAGENET_DIR)/train*',
+      evalFilePattern: '$(IMAGENET_DIR)/valid*',
+    },
+  },
+  coco:: {
+    scriptConfig+: {
+      trainFilePattern: '$(COCO_DIR)/train*',
+      evalFilePattern: '$(COCO_DIR)/val*',
+      paramsOverride+: {
+        task+: {
+          annotation_file: '$(COCO_DIR)/instances_val2017.json',
+        },
+      },
+    },
+  },
+  local functional_schedule = '0 9 * * *',
+  Functional:: mixins.Functional {
+    schedule: if !(self.accelerator.type == 'tpu') || self.accelerator.name == 'v3-8' || self.accelerator.name == 'v4-8' then
+      functional_schedule
+    else
+      null,
+    metricConfig+: {
+      sourceMap+:: {
+        tensorboard+: {
+          aggregateAssertionsMap+:: {
+            examples_per_second: {
+              AVERAGE: {
+                inclusive_bounds: true,
+                std_devs_from_mean: {
+                  comparison: 'GREATER',
+                  std_devs: 4.0,
+                },
+                wait_for_n_data_points: 0,
+              },
+            },
+          },
+        },
+      },
+    },
+  },
+  // Override default schedule for Functional.
+  RunNightly:: {
+    schedule: functional_schedule,
+  },
+  Convergence:: mixins.Convergence {
+    schedule: '0 5 * * 0,2,4',
```

Comment:
Ditto, run on other days so that we don't conflict with nightly tests. Make sure it runs at the immediate possible time.

Reply:
Would `0 0 * * 0,2,4` work or should I do `0 5 * * 1,3,5` (I suppose if the convergence tests take <5 hours the first should be fine).